mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -76,6 +76,24 @@ class LLVM_IntArithmeticOpWithOverflowFlag<string mnemonic, string instName,
     "$res = builder.Create" # instName #
     "($lhs, $rhs, /*Name=*/\"\", op.hasNoUnsignedWrap(), op.hasNoSignedWrap());";
 }
+class LLVM_IntArithmeticOpWithIsExact<string mnemonic, string instName,
```

Comment:
```suggestion
class LLVM_IntArithmeticOpWithExactFlag<string mnemonic, string instName,
```
nit: Should we use ExactFlag to follow the naming of LLVM_IntArithmeticOpWithIsExact?

Reply:
Done!